src/background/cache.js

```diff
@@ -0,0 +1,240 @@
+const crypto = require('crypto')
```

Comment:
TODO: Split this file out into it's own module (from express-asset-file-cache-middleware)